Slugifier/SlugifierInterface.php

```diff
@@ -11,19 +11,12 @@
 
 namespace Symfony\Cmf\Bundle\CoreBundle\Slugifier;
 
+use Symfony\Cmf\Api\Slugifier\SlugifierInterface as ApiSlugifierInterface;
+
 /**
- * Slugifier interface.
- *
- * @author Daniel Leech <daniel@dantleech.com>
+ * @deprecated Since version 1.3, to be removed in 2.0. Install and use the
+ *             symfony-cmf/slugifier-api package instead.
  */
-interface SlugifierInterface
+interface SlugifierInterface extends ApiSlugifierInterface
```

Comment:
To be honest, this does not make sense at all: Interfaces are there to typehint, if one would typehint for the interface in the CoreBundle and get the CallbackSlugifier from the slugifier-api package, an error will appear on the screen.

In order to be BC, the interface in slugifier-api has to extend this one (the old one). However, in that case, the slugifier-api package has to require the CoreBundle and all advantages of a separate package are gone.

What should we do here?

 1. Remove everything from this bundle, documenting the BC break in UPGRADE-1.3.md and CHANGELOG.md
 1. Keep it as it is